__mocks__/readme.md

```diff
@@ -0,0 +1,2 @@
+Install Camouflage: npm install -g camouflage-server
+Start the Camouflage server by initializing it with a config.yml file (): camouflage -c ./config.yml (Ensure that you are in the __mocks__ directory before running the command)
```

Comment:
this is really great now.
you only need to use the actual Markdown syntax for readme file to properly format it.
here is the reference: [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)